src/zenml/integrations/aws/flavors/aws_container_registry_flavor.py

```diff
@@ -73,7 +73,24 @@ def docs_url(self) -> Optional[str]:
         Returns:
             A flavor docs url.
         """
-        return self.generate_default_docs_url()
+        from packaging.version import parse
+
+        from zenml import __version__
+
+        if parse(__version__) >= parse("0.34.0"):
+            return self.generate_default_docs_url()
+
+        old_docs_name = "amazon-ecr"
+        return self.generate_default_docs_url(component_name=old_docs_name)
+
+    @property
+    def sdk_docs_url(self) -> Optional[str]:
```

Comment:
Maybe im missing something but if this function exists in all flavors then why cant we just put it in the BaseFlavor class?

Reply:
Same was true for the `docs_url`, but that also wasn't in the `BaseFlavor`. You're right it probably should be, though.

Reply:
This is true for our flavors, but not custom user flavors which subclass the same`Flavor` base class